Assets/Scripts/PopulationController.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using Database.Tables;
+using DataStructures;
+
+
+public class PopulationController : MonoBehaviour, IPointerClickHandler
+{    
+    public DatabaseManager _databaseManager;
+    private Canvas _canvas;
```

Comment:
```suggestion
    [SerializeField]
    private Canvas _canvas;
```

This will expose the _canvas in the inspector for the Populations game object. We can then drag and drop the game object containing the canvas to get the reference.